src/components/Gradebook/footer.jsx

```diff
@@ -0,0 +1,89 @@
+              <li><a href="https://support.edx.org">Help Center</a></li>
+              <li><a href="https://www.edx.org/media-kit">Media Kit</a></li>
+              <li><a href="https://www.edx.org/donate">Donate</a></li>
+            </ul>
+          </div>
+          <div className="area-5">
+            <ul className="d-flex flex-row justify-content-between list-unstyled max-width-222 mb-4">
+              {/* TODO: Use Paragon HyperLink with Icon. Would need to add rel to paragon if we still need it. */}
+              <li><a href="http://www.facebook.com/EdxOnline" title="Facebook" rel="noreferrer" target="_blank">
+                <Icon className={['fa', 'fa-facebook-square', 'fa-2x']} screenReaderText="Like edX on Facebook" />
+              </a></li>
+              <li><a href="https://twitter.com/edXOnline" title="Twitter" rel="noreferrer" target="_blank">
+                <Icon className={['fa', 'fa-twitter-square', 'fa-2x']} screenReaderText="Follow edX on Twitter" />
+              </a></li>
+              <li><a href="https://www.youtube.com/user/edxonline" title="Youtube" rel="noreferrer" target="_blank">
+                <Icon className={['fa', 'fa-youtube-square', 'fa-2x']} screenReaderText="Subscribe to the edX YouTube channel" />
+              </a></li>
+              <li><a href="https://www.linkedin.com/company/edx" title="LinkedIn" rel="noreferrer" target="_blank">
+                <Icon className={['fa', 'fa-linkedin-square', 'fa-2x']} screenReaderText="Follow edX on LinkedIn" />
+              </a></li>
+              <li><a href="https://plus.google.com/+edXOnline" title="Google+" rel="noreferrer" target="_blank">
+                <Icon className={['fa', 'fa-google-plus-square', 'fa-2x']} screenReaderText="Follow edX on Google+" />
+              </a></li>
+              <li><a href="https://www.reddit.com/r/edx" title="Reddit" rel="noreferrer" target="_blank">
+                <Icon className={['fa', 'fa-reddit-square', 'fa-2x']} screenReaderText="Subscribe to the edX subreddit" />
+              </a></li>
+            </ul>
+            <ul className="d-flex flex-row justify-content-between list-unstyled max-width-264 mb-5">
+              <li><a href="https://itunes.apple.com/us/app/edx/id945480667?mt=8" target="_blank">
+                <img className="max-height-39"
+                     alt="Download the edX mobile app from the Apple App Store"
+                     src="https://prod-edxapp.edx-cdn.org/static/images/app/app_store_badge_135x40.d0558d910630.svg" />
+              </a></li>
+              <li><a href="https://play.google.com/store/apps/details?id=org.edx.mobile" target="_blank">
+                <img className="max-height-39"
+                     alt="Download the edX mobile app from Google Play"
+                     src="https://prod-edxapp.edx-cdn.org/static/images/app/google_play_badge_45.6ea466e328da.png" />
+              </a></li>
```

Comment:
There is some weird formatting going on in the file, it would be great to add the edX JS linter to this repo for consistency (https://github.com/edx/eslint-config-edx).